docs/src/components/home/StartToday.tsx

```diff
@@ -57,8 +57,10 @@ export default function StartToday() {
           <Paper
             component={Link}
             href={ROUTES.blog}
-            target="_blank"
-            rel="noreferrer noopener"
+            {...(ROUTES.blog.startsWith('http') && {
+              target: '_blank',
+              rel: 'noreferrer noopener',
+            })}
```

Comment:
What the purpose of this change?

Reply:
`ROUTES.blog` before the toggle turn on point to `https:medium.com`. Now, with `/blog` the `target: '_blank'` is not needed. I think I will just remove it then.